src/angularComponent/ngcOmniboxController.js

```diff
@@ -240,10 +242,14 @@ export default class NgcOmniboxController {
    * @param {Object} item
    */
   unchoose(item) {
-    if (item && Array.isArray(this.ngModel)) {
-      this.ngModel.splice(this.ngModel.indexOf(item), 1);
-    } else if (!this.multiple) {
-      this.ngModel = null;
+    if (item) {
+      if (Array.isArray(this.ngModel)) {
+        this.ngModel.splice(this.ngModel.indexOf(item), 1);
```

Comment:
by "work" do you mean it won't throw an error? I think indexOf would just return -1, which I guess is harmless, but I also don't see the harm in bailing if no item is passed.